pyproject.toml

```diff
@@ -54,9 +54,9 @@ dependencies = [
 
 [project.optional-dependencies]
 tf = [
-    "tensorflow>=2.9.0,<3.0.0",  # cf. https://github.com/mindee/doctr/issues/454
+    "tensorflow>=2.11.0,<3.0.0",  # cf. https://github.com/mindee/doctr/issues/454
```

Comment:
Open question: should we update the comment to reflect what we aim to fix with that?

I've already found it much more useful than I anticipated to have the references to the multiple issues that led to choosing the version specifiers.

Reply:
added ref to PR :)